Assets/Editor/CreatureHydrator.cs

```diff
@@ -0,0 +1,263 @@
+using UnityEngine;
+using UnityEditor;
+using UnityEditor.Animations;
+using YamlDotNet.Serialization;
+using YamlDotNet.Serialization.NamingConventions;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+public class CreatureHydrator : Editor
+{
+    [MenuItem("Tools/Hydrate Creatures")]
+    public static void HydrateCreatures()
+    {
+        string rootGeneraPath = "Assets/Creatures/impl";
+        string[] generaDirectories = Directory.GetDirectories(rootGeneraPath);
+
+        foreach (var generaDir in generaDirectories)
+        {
+            string generaName = Path.GetFileName(generaDir);
```

Comment:
personal pref, make this block a function, and just have `return` instead of `continue`s

Reply:
agreed! Done 👌